pkg/addons2/velero/velero.go

```diff
@@ -11,7 +11,8 @@ import (
 )
 
 type Velero struct {
-	Proxy *ecv1beta1.ProxySpec
+	Proxy    *ecv1beta1.ProxySpec
+	IsAirgap bool
```

Comment:
why is this unused?

Reply:
well that is certainly an argument in favor of centralizing the 'should this be airgap mode' flag

Reply:
I've changed install2 to pass the helm client as a param, so airgap logic gets removed from the addon code